types/ember/index.d.ts

```diff
@@ -3528,7 +3528,8 @@ declare module '@ember/object' {
 
 declare module '@ember/object/computed' {
     import Ember from 'ember';
-    export default class ComputedProperty<T> extends Ember.ComputedProperty<T> { }
+    type Computed<T> = Ember.ComputedProperty<T>;
+    export default Computed;
```

Comment:
Besides the issue of IDE behavior, I think this actually ends up making this *not* work if someone actually *does* need to use `ComputedProperty` as a `class` and not just a `type` alias. We need the default export to be a class for that reason, I believe? 😬